loopd/swapclient_server.go

```diff
@@ -208,6 +208,18 @@ func (s *swapClientServer) LoopOut(ctx context.Context,
 		PaymentTimeout:          paymentTimeout,
 	}
 
+	// If the asset id is set, we need to set the asset amount and asset id
+	// in the request.
+	if in.AssetId != nil {
+		req.AssetAmount = btcutil.Amount(in.Amt)
+		req.AssetId = in.AssetId
+		req.AssetEdgeNode = in.AssetEdgeNode
```

Comment:
I think we should enforce that if asset_id is set, the asset edge node is also set, as specified in the proto.